### PR TITLE
Interactivity API: Move all utils inside `utils.ts`

### PR DESCRIPTION
**Pull request by DAreRodz**
## What?

Right now, utility functions live in different files and folders. This PR moves everything inside the `utils.ts` file, just to keep the code a bit more organized.

### CODE REVIEW

**Review by sirreal — COMMENTED** [review]
It's kind of weird to have the directory, some single-function modules, and the index module with a bunch of stuff. What if we flatten and make it all `utils.ts`?

**Comment by DAreRodz** [comment]
> What if we flatten and make it all `utils.ts`?

Sure. Let's do that.

**Review by cbravobernal — APPROVED** [review]
LGTM

File: test/unit/jest.config.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ module.exports = {
 		'^.+\\.[jt]sx?$': '<rootDir>/test/unit/scripts/babel-transformer.js',
 	},
 	transformIgnorePatterns: [
-		'/node_modules/(?!(docker-compose|yaml)/)',
+		'/node_modules/(?!(docker-compose|yaml|preact|@preact|deepsignal)/)',
```
Comment (DAreRodz) [inline test/unit/jest.config.js:43]: Thank you, @sirreal! 🙌

**Pull request merged**